docs/UserGuide.md

```diff
@@ -133,30 +133,43 @@ Shows a list of all the existing tasks in the task list.
 
 Format: `list`
 
-### Deleting a task: `delete`
+### 3.3 [Coming soon] Listing all tags: `list t/`
 
-Deletes a specified task from Harmonia.
+Lists all existing tags used in the task list.
+
+Format: `list t/`
+
+### 3.4 Deleting a task: `delete`
+
+Deletes the task at the given index of the existing task list from Harmonia.
 
 Format: `delete INDEX`
-- Deletes the task at the specified `INDEX` from the task list.
-- The index refers to the index number shown in the displayed task list.
-- The index **must be a positive integer** 1, 2, 3
 
-Example: `delete 3` deletes the 3rd task in the task list
+Example: `delete 3` deletes the 3rd task in Harmonia.
 
-### Locating a task: `find`
+<div markdown="span" class="alert alert-primary">:bulb: **Note:**
+You can key list to see the sequence of tasks in the list to check the index of the task you wish to delete.
+</div>
```

Comment:
Should 'list' be formatted as `list` (i.e. You can key `list` to see the sequence of tasks...)?

Reply:
Thanks for catching this! Will fix soon.